hedera-node/hedera-app/src/main/java/com/hedera/node/app/authorization/PrivilegesVerifier.java

```diff
@@ -86,22 +83,6 @@ public SystemPrivilege hasPrivileges(
                     payerId, txBody.fileUpdateOrThrow().fileIDOrThrow().fileNum());
             case FILE_APPEND -> checkFileChange(
                     payerId, txBody.fileAppendOrThrow().fileIDOrThrow().fileNum());
-            case CONTRACT_UPDATE -> {
```

Comment:
Why are `CONTRACT_UPDATE` and `CONTRACT_DELETE` removed?

Reply:
There are no system contracts right now. This is confusing to think we need privileges for `CONTRACT_UPDATE` and `CONTRACT_DELETE`. @tinker-michaelj  had a comment[ on the PR](https://github.com/hashgraph/hedera-services/pull/9151#discussion_r1367152481) , but it was not removed.